app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.kt

```diff
@@ -137,16 +61,43 @@ class ThirdPartyTransferFragment : BaseFragment(), ThirdPartyTransferView, OnIte
             savedInstanceState: Bundle?
     ): View? {
         (activity as BaseActivity?)?.activityComponent?.inject(this)
-        rootView = inflater.inflate(R.layout.fragment_third_party_transfer, container, false)
+        errorLayoutBinding = ErrorLayoutBinding.inflate(inflater,container,false)
+        _binding = FragmentThirdPartyTransferBinding.inflate(inflater,container,false)
         setToolbarTitle(getString(R.string.third_party_transfer))
-        if (rootView != null ) ButterKnife.bind(this, rootView!!)
-        sweetUIErrorHandler = SweetUIErrorHandler(activity, rootView)
+        sweetUIErrorHandler = SweetUIErrorHandler(activity, binding.root)
         showUserInterface()
         presenter?.attachView(this)
         if (savedInstanceState == null) {
             presenter?.loadTransferTemplate()
         }
-        return rootView
+
+        binding.btnReviewTransfer.setOnClickListener {
+            reviewTransfer()
+        }
+
+        binding.btnPayFrom.setOnClickListener {
+            payFromSelected()
+        }
+
+        binding.btnPayTo.setOnClickListener {
+            payToSelected()
+        }
+
+        binding.btnAmount.setOnClickListener {
+            amountSet()
+        }
+
+        binding.btnCancelTransfer.setOnClickListener {
+            cancelTransfer()
+        }
+
+        binding.btnAddBeneficiary.setOnClickListener {
+            addBeneficiary()
+        }
+        errorLayoutBinding.btnTryAgain.setOnClickListener {
```

Comment:
We can access this using binding.layoutError.btnTryAgain(do a cross check). I think using a separate errorLayoutBinding variable is not required.

Reply:
Also in the terms of context referencing "btnTryAgain" using errorLayoutBinding will be different than using binding. layoutError.btnTryAgain

Reply:
True that , I will try this and let you know